.gitignore

```
@@ -112,3 +112,6 @@ ENV/
 metadata.csv
 /data/
 db.sqlite3
+node_modules
+*.DS_Store
+*/.DS_Store
```

```suggestion
.DS_Store
```

(see: https://github.com/github/gitignore/blob/master/Global/macOS.gitignore)